juriscraper/opinions/united_states/state/conn.py

```diff
@@ -65,28 +63,35 @@ def _extract_cases_from_html(self, html):
                         anchor = element
                         glued = f"{anchor.text_content()} {anchor.tail}"
                         text = normalize_dashes(" ".join(glued.split()))
+                    docket = (
+                        text.split("-")[0]
+                        .split("Concurrence")[0]
+                        .split("Dissent")[0]
+                        .split("Appendix")[0]
+                    )
+                    try:
+                        name = text.split("-", 1)[1]
+                    except IndexError:
+                        # Expected link text:
+                        # - "AC44191 Dissent Marshall v. Commissioner of Motor Vehicles"
+                        try:
+                            name = text.split("Dissent", 1)[1]
+                        except IndexError:
+                            try:
+                                name = text.split("Concurrence", 1)[1]
+                            except IndexError:
+                                try:
+                                    name = text.split("Appendix", 1)[1]
+                                except:
+                                    logger.info(
+                                        f"Name not found for case: '{text}'"
```

Comment:
this smells ... to me.

Reply:
I also don't like it :-(
Do I change it a `for` loop?

Reply:
docket_regex would vary between supreme and appellate. but its really simple 

`self.docket_regex = r"SC\d+"`

or 

` self.docket_regex = r"AC\d+"`

Reply:
also - if this was the approach to take, which It certainly could be - I dont think throwing info at the end is what we want.  I think we would want to throw a warning so we can discover why and account for it in the future.

Reply:
Added RE to serach for the dockets from each court.